Packages/com.llealloo.audiolink/Runtime/Scripts/ThemeColorController.cs

```diff
@@ -16,13 +16,34 @@ public class ThemeColorController : MonoBehaviour
     {
         [UdonSynced] private int _themeColorMode;
 
+        public Color[] customThemeColors
+        {
+            get
+            {
+                return new[] {
```

Comment:
somethings gone funky with the indentation here I think

Reply:
in what way? not sure I understand